subprojects/platform-native/src/testFixtures/groovy/org/gradle/nativeplatform/fixtures/app/CppGreeterUsingCoreFoundation.groovy

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2018 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.nativeplatform.fixtures.app
+
+import static org.gradle.nativeplatform.fixtures.app.SourceFileElement.ofFile
+
+class CppGreeterUsingCoreFoundation extends CppLibraryElement implements GreeterElement {
```

Comment:
this and the following fixture is only used by one test, I think it would be a lot simpler just to inline the source file into the test, like the new tests for the C and C++ software model plugins.

Reply:
The code was inlined.